### PR TITLE
Update "Create Your First Service" guide to reflect the latest UI changes

### PR DESCRIPTION
**Pull request by RukshiW**

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
+/workspace.xml
```
Comment (anupama-pathirage) [inline .idea/.gitignore:3]: Remove these .idea files.

**Comment by RukshiW** [comment]
Corrects https://github.com/wso2-enterprise/choreo/issues/3837

**Comment by RukshiW** [comment]
Corrects https://github.com/wso2-enterprise/choreo/issues/3845

File: .idea/.gitignore (hunk 1 above)
Comment (RukshiW) [reply .idea/.gitignore]: Removed

File: en/docs/services/create-your-first-service.md [hunk 2]
```diff
@@ -22,66 +22,71 @@ To create the Choreo service, follow the procedure below:
     
     Then click **Save API**. Now you have saved the API trigger that starts your application.
     
-5. In the next form that appears, do as follows to configure the HTTP connection that connects the public API used in this scenario to the API trigger you configured.
+5. Click **API Calls**, and then enter informationas follows to configure the HTTP connection that connects the public API used in this scenario to the API trigger you configured.
```
Comment (anupama-pathirage) [inline en/docs/services/create-your-first-service.md:25]:
```suggestion
5. Click **API Calls**, and then enter information as follows to configure the HTTP connection that connects the public API used in this scenario to the API trigger you configured.
```

**Comment by RukshiW** [comment]
Corrects https://github.com/wso2-enterprise/choreo/issues/3897

**Comment by RukshiW** [comment]
Closing this PR because it shows conflicts that cannot be viewed or corrected.

**Pull request closed without merging**